backend/src/service/user-service.js

```diff
@@ -1,24 +1,55 @@
 import { getRepository } from "typeorm";
 import { User } from "../model/user";
+import { crypto } from "../common/lib";
 
 class UserService {
```

Comment:
```suggestion
class UserService {
    static instance = null;
    static getInstance() {
        if (UserService.instance === null) {
            UserService.instance = new UserService();
        }
        return UserService.instance;
    }
```
싱글톤 패턴은 어떨까요? babel을 써서 static property가 지원됩니다!

Reply:
싱글톤 방식으로 수정했습니다 :-)